NiFiDeploy.groovy

```diff
@@ -40,6 +42,14 @@ cli.with {
     args:1, argName:'uri', type:String.class
   c longOpt: 'client-id', 'Client ID for API calls, any unique string (override)',
     args:1, argName:'id', type:String.class
+  k longOpt: 'keystore', 'The keystore file for ssl',
+    args:1, argName:'keystore', type:String.class
+  p longOpt: 'keypasswd', 'The password for the keystore',
+    args:1, argName:'keypasswd', type:String.class
+  u longOpt: 'truststore', 'The keystore file for ssl',
+    args:1, argName:'truststore', type:String.class
+  r longOpt: 'trustpasswd', 'The password for the truststore',
+    args:1, argName:'trustpasswd', type:String.class
```

Comment:
These `keypasswd` and `trustpasswd` long option names are too prone to typos, IMO. I understand why you don't list 'password' anywhere, so maybe let's reduce them to `keypass` and `trustpass`?